openssl-sys/build/main.rs

```diff
@@ -221,6 +221,8 @@ See rust-openssl README for more information:
             (3, 1, 0) => ('3', '1', '0'),
             (3, 1, _) => ('3', '1', 'x'),
             (3, 2, 0) => ('3', '2', '0'),
+            (3, 2, 1) => ('3', '2', '1'),
+            (3, 2, 2) => ('3', '2', '2'),
```

Comment:
Since 3.2.2 is the first stable release of the 3.2.x series, you can allow all `3.2.x` versions here like the `(3, 1, _)` line above.